creator-node/src/utils.js

```diff
@@ -196,6 +202,84 @@ const ipfsGet = (path, req, timeout = 1000) => new Promise(async (resolve, rejec
   }
 })
 
+async function findCIDInNetwork (filePath, cid, logger) {
+  const attemptedStateFix = await getIfAttemptedStateFix(filePath)
+  if (attemptedStateFix) return
+
+  // generate signature
+  const delegateWallet = config.get('delegateOwnerWallet').toLowerCase()
+  const { signature, timestamp } = generateTimestampAndSignature({ filePath, delegateWallet }, config.get('delegatePrivateKey'))
+  const creatorNodes = await getAllRegisteredCNodes()
+  let node
+
+  for (let index = 0; index < creatorNodes.length; index++) {
+    node = creatorNodes[index]
+    try {
+      const resp = await axios({
+        method: 'get',
+        url: `${node.endpoint}/file_lookup`,
+        params: {
+          filePath,
+          timestamp,
+          delegateWallet,
+          signature
+        },
+        responseType: 'stream',
+        timeout: 1000
```

Comment:
do we know how long it takes to stream, eg, a transcode file for a massive set mix? not sure if 1s timeout is sufficient

Reply:
yeah so this is a response timeout. it doesn't expect the whole file to be downloaded in 1 second. only if there's no response data at all, then it will timeout. if the pipe is slow but it's sending data it can take arbitrarily long